### ISSUE TITLE
 use it in Salmonella

### ISSUE THREAD

**Issue by fph345** [jmarshallnz/islandR#25]
hello, I have two questions to ask. Your package uses an AI model with covariates. Can the version without covariates be run through by setting parameters? Can I use it in Salmonella ,being compatible with Salmonella's housekeeping genes.
thank you！

**Comment by jmarshallnz** [comment]
Yes, that should be fine. To run an attribution without covariates just use `ST ~ 1` as the formula in `attribution`.

**Comment by fph345** [comment]
Thank you for your reply.Could you provide further details when I use st_fit() in Salmonella,It seems that the sequences parameter for Salmonella is not supported. How can I modify the code.
![12](https://github.com/jmarshallnz/islandR/assets/139038385/a3eec3a5-f423-4d15-9c19-d75ab22d21d5)
